components/com_kunena/site/template/crypsis/media/js/upload.main.js

```diff
@@ -214,7 +214,12 @@ jQuery(function($) {
 				if($.isEmptyObject(data.files)==false) {
 					fileCount = Object.keys(data.files).length;
 					$( data.files ).each(function( index, file ) {
-						var object = $( '<div><p><img src="'+file.path+'" width="100" height="100" /><br /><span>'+file.name+'</span><br /></p></div>' );
+						var image = '';
+						if (file.image===true) {
+							image = '<img src="'+file.path+'" width="100" height="100" /><br />';
```

Comment:
If it's an image it display the image thumbnail else it display nothing, so what is wrong ?